### DISCUSSION TITLE
linux_x2 node environment v18.19.0 install node module awaly timeout, but mac is ok

### DISCUSSION THREAD

**Discussion by sonorus-eric** [codemagic-ci-cd/codemagic-docs#2303] [Q&A] [answered]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

React Native

### Steps to reproduce

![image](https://github.com/codemagic-ci-cd/codemagic-docs/assets/115517910/f9953729-b30b-46bd-b6c6-eafbaae80a16)

### Expected results

install  dependencies success

### Actual results

install node modules awaly timeout

set mechine is mac_mini_m1 is ok, but set linux is still error

![image](https://github.com/codemagic-ci-cd/codemagic-docs/assets/115517910/ae64a289-75b4-41f0-96c9-940e883395f3)

**Comment by himesh-cm** [comment, marked as answer]
Hi @sonorus-eric,
After testing, I found that this issue is specific to version `18.19.0` in a Linux environment. You can refer to the GitHub issue for more information: https://github.com/npm/cli/issues/4028. 

To resolve the issue, please try using node 20.11.0.

**Reply by sonorus-eric** [reply]
Thank you reply!  is ok
